vite.config.js

```diff
@@ -3,8 +3,8 @@ import { viteStaticCopy } from "vite-plugin-static-copy";
 
 const cesiumSource = "node_modules/cesium/Build/Cesium";
 // This is the base url for static files that CesiumJS needs to load.
-// Set to an empty string to place the files at the site's root path
-const cesiumBaseUrl = "cesiumStatic";
+// Set to "/" to place the files at the site's root path
+const cesiumBaseUrl = "/cesiumStatic";
```

Comment:
`npm run build` will not work with this. It will try and put the static files at the root of your computer which it usually doesn't have access too and would be bad practice even if it did.

The real problem is the value that `CESIUM_BASE_URL` is set to not where the files are output too. I think changing that to `JSON.stringify(`/${cesiumBaseUrl}`),` instead so it's always using the root but `viteStaticCopy` isn't trying to access the root is the more correct change. Can you test that in your other branch and update the PR if it's still working as I think it should?

Reply:
Good catch! I tried that change out and it still worked. It's definitely the more correct change to make.

Vite mainly supports SPAs, but those sometimes have multiple routes if they use something like vue-router.